app/models/team.rb

```diff
@@ -0,0 +1,10 @@
+class Team
+  include Mongoid::Document
+
+  field :winner, type: String
```

Comment:
you should validate that this is in the expected list. e.g it's one of 'team_one', 'team_two', and 'tie' or whatever